testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/core/basic/DisabledDefaultExceptionThrowableMapperMapperTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ *
+ * Copyright 2022 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jboss.resteasy.test.core.basic;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+import jakarta.enterprise.context.RequestScoped;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.ext.ExceptionMapper;
+import jakarta.ws.rs.ext.Provider;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.resteasy.test.core.basic.resource.ExceptionResource;
+import org.jboss.resteasy.utils.TestUtil;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * Tests that the default {@link ExceptionMapper} is disabled.
+ *
+ * @author <a href="mailto:jperkins@redhat.com">James R. Perkins</a>
+ */
+@RunWith(Arquillian.class)
+@RequestScoped
+public class DisabledDefaultExceptionThrowableMapperMapperTest extends DisabledDefaultExceptionMapperTest {
+
+    @Deployment
+    public static WebArchive createDeployment() {
+        return ShrinkWrap.create(WebArchive.class, DisabledDefaultExceptionThrowableMapperMapperTest.class.getSimpleName() + ".war")
+                .addClasses(
+                        DisabledDefaultExceptionMapperTest.class,
+                        ExceptionResource.class,
+                        TestUtil.class
+                )
+                .addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml");
+    }
+
+    /**
+     * Tests that the default exception mapper does not exist.
+     */
+    @Test
+    public void defaultExceptionMapper() {
+        final ExceptionMapper<?> mapper = providers.getExceptionMapper(RuntimeException.class);
+        Assert.assertNotNull("Expected expected our default exception mapper", mapper);
+        Assert.assertTrue(String.format("Expected mapper %s to be instance of %s.", mapper, ThrowableExceptionMapper.class),
+                mapper instanceof ThrowableExceptionMapper);
```

Comment:
Wouldn't this be the case even if the DefaultExceptionMapper were enabled, given that ThrowableExceptionMapper isn't builtin?

Reply:
Yes, it would. I guess I kept it to just ensure we get the correct one.